src/lib/movies.ts

```diff
@@ -106,6 +106,29 @@ export const getMovies = unstable_cache(
   { revalidate: 1 },
 );
 
+export const getTheaters = unstable_cache(
```

Comment:
I wonder if we should put this in a separate file called `src/lib/theaters.ts`

Reply:
Yes agreed. Done